docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -0,0 +1,330 @@
+- Access to a storage bucket like Amazon S3, Azure Blob Storage, or Google Cloud Platform (for example, [Google Cloud Storage instructions](https://cloud.google.com/storage/docs/creating-buckets))
+- DNS service works in the Kubernetes cluster
+- Optional: An ingress controller is set up in the Kubernetes cluster, for example [ingress-nginx](https://kubernetes.github.io/ingress-nginx/)
+
+>**NOTE**: If you want to access Tempo from outside of the Kubernetes cluster, you will need an ingress. Ingress-related procedures are marked as optional.
+
+<!-- This section should be verified before being made visible. It’s from Mimir and might need to be updated for Tempo.
+
+## Security setup
+
+This installation will not succeed if you have enabled the [PodSecurityPolicy](*https://v1-23.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) admission controller or if you are enforcing the Restricted policy with [Pod Security](https://v1-24.docs.kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces) admission controller. The reason is that the installation includes a deployment of MinIO. The [minio/minio chart](https://github.com/minio/minio/tree/master/helm/minio) is not compatible with running under a Restricted policy or the PodSecurityPolicy that the mimir-distributed chart provides.
+
+If you are using the PodSecurityPolicy admission controller, then it is not possible to deploy the mimir-distributed chart with MinIO. Refer to Run Grafana Mimir in production using the Helm chart for instructions on setting up an external object storage and disable the built-in MinIO deployment with minio.enabled: false in the Helm values file.
+
+If you are using the Pod Security admission controller, then MinIO and the tempo-distributed chart can successfully deploy under the baseline pod security level.
+-->
+
+## Create a custom namespace and add the Helm repository
+
+Using a custom namespace solves problems later on because you do not have to overwrite the default namespace.
+
+1. Create a unique Kubernetes namespace, for example tempo-test:
+
+```bash
+kubectl create namespace tempo-test
+```
+
+   For more details, see the Kubernetes documentation about [Creating a new namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
+
+1. Set up a Helm repository using the following commands:
+
+   ```bash
+   helm repo add grafana https://grafana.github.io/helm-charts
+   helm repo update
+   ```
+
+   >**NOTE**: The Helm chart at https://grafana.github.io/helm-charts is a publication of the source code at grafana/tempo.
+
+
+## Set Helm chart values
+
+Your Helm chart values are set in the `custom.yaml` file. The following example `custom.yaml` file sets the storage and traces options, enables the gateway, and sets the cluster to main. The `traces` configure the distributor's receiver protocols.
```

Comment:
@zalegrala I referenced the GET Helm chart doc and we point people to this file: https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/values.yaml 

Should we be using that values.yaml file instead of creating a custom.yaml file? The values.yaml file has the section for MinIO already there. We'd just have to tell people to change the value to enabled.